NTPClient.h

```diff
@@ -15,7 +15,7 @@ class NTPClient {
 
     const char*   _poolServerName = "pool.ntp.org"; // Default time server
     IPAddress     _poolServerIP;
-    int           _port           = NTP_DEFAULT_LOCAL_PORT;
+    long           _port           = NTP_DEFAULT_LOCAL_PORT;
```

Comment:
Hi @luigigubello :wave: :coffee: Is there any special reason why you change this data type from `int` to `long`? Certainly `int` is capable of holding values up 2^16-1 even on AVR :wink:

Reply:
> Certainly int is capable of holding values up 2^16-1 even on AVR

[Reading the documentation](https://www.arduino.cc/reference/en/language/variables/data-types/int/) I have understood it's not, but your suggestion to use `unsigned int` is a good idea. So, yes, I have chosen `long` instead `int` to be sure to maintain the AVR compatibility. It seems `unsigned int` is the best choice 🙂

Reply:
I've checked again - so yes, `int` [2 byte](https://gcc.gnu.org/wiki/avr-gcc) on AVR which covers from -2^15 to 2^15-1. `unsigned int` covers 0 to 2^16-1 which is enough.